views/404.slim

```diff
@@ -132,23 +134,28 @@ javascript:
     var pathbits = path.split("/")
     var reparse = document.getElementById("reparse")
 
-    if (pathbits.length > 2) {
-      var script = document.createElement( "script" );
-      var address = "http://199.229.252.196:4567/redeploy/" + pathbits[2]
-
-      if (pathbits.length > 3 && pathbits[3] != "") address += "/" + pathbits[3];
-      if (pathbits.length == 3) address += "/latest"
-      
-      script.type = "text/javascript";
-      script.src = address
-      document.head.appendChild(script);
-
-      reparse.innerHTML = "Thanks, reparsing right now. Please give it some time and come back in 5-10 mins."
-      reparse.disabled = true
-    } else {
-
+    if (pathbits.length <= 2) {
       reparse.innerHTML = "Could not find the Podname from your address, please check a docs full URL."
     }
+
+    var address = "http://199.229.252.196:4567/redeploy/" + pathbits[2]
```

Comment:
Is there a DNS entry for this server? api.cocoadocs.org does not resolve to this IP oddly enough.